src/components/Patient/PatientDetailsTab/Demography.tsx

```
@@ -38,6 +39,27 @@
+  md.renderer.rules.link_open = (tokens, idx) => {
+    const token = tokens[idx];
+    const href = token.attrGet("href");
+    return `<a href="${href}" target="_blank" rel="noopener noreferrer" class="text-blue-600 hover:text-blue-800 underline break-all">`;
+  };
+
+  // Simple utility function to render text with clickable URLs using markdown-it
+  const renderTextWithLinks = (text: string) => {
+    if (!text) return "";
+
+    const rendered = md.renderInline(text);
+    return <span dangerouslySetInnerHTML={{ __html: rendered }} />;
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)